### PR TITLE
[5.5] update dissect filter and other plugins to latest patch version

### PR DESCRIPTION
**Pull request by suyograo**
and regenerate the lock file to get other patch releases for plugins.

Related to https://github.com/logstash-plugins/logstash-filter-dissect/pull/32

### CODE REVIEW

File: Gemfile.template [hunk 1]
```diff
@@ -42,7 +42,7 @@ gem "logstash-codec-json_lines"
 gem "logstash-codec-line"
 gem "logstash-codec-msgpack"
 gem "logstash-codec-multiline"
-gem "logstash-codec-netflow"
+gem "logstash-codec-netflow", "~> 3.4.0"
```
Comment (suyograo) [inline Gemfile.template]: I had to add these to not automatically fetch minor versions for these plugins.

**Comment by colinsurprenant** [comment]
do we want to restrict the change to only the dissect filter or we are ok updating all plugins to latest in a bug release?

**Comment by suyograo** [comment]
@colinsurprenant as long as they are all patch fixes in the plugins, we should be good. We've been doing this for all of Logstash's patch fixes in 5.0

**Comment by suyograo** [comment]
Changed the commit/title to reflect that we're updating more than dissect filter.

File: Gemfile.jruby-1.9.lock.release [hunk 2]
```diff
@@ -129,10 +129,10 @@ GEM
     jruby-openssl (0.9.19-java)
     jruby-stdin-channel (0.2.0-java)
     json (1.8.6-java)
-    kramdown (1.13.2)
+    kramdown (1.14.0)
```
Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock.release]: 1.13.2 -> 1.14.0  ok?

File: Gemfile.jruby-1.9.lock.release [hunk 3]
```diff
@@ -535,8 +535,8 @@ GEM
     rack (1.6.6)
     rack-protection (1.5.3)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
+    rack-test (0.7.0)
```
Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock.release]: 0.6.3 -> 0.7.0 ok?

File: Gemfile.jruby-1.9.lock.release [hunk 4]
```diff
@@ -567,7 +567,7 @@ GEM
     sawyer (0.6.0)
       addressable (~> 2.3.5)
       faraday (~> 0.8, < 0.10)
-    sequel (4.47.0)
+    sequel (4.48.0)
```
Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock.release]: 4.47.0 -> 4.48.0 ok?

**Comment by colinsurprenant** [comment]
few comments to some minor version upgrades vs patch version.

File: Gemfile.jruby-1.9.lock.release (hunk 2 above)
Comment (suyograo) [inline Gemfile.jruby-1.9.lock.release]: yep

File: Gemfile.jruby-1.9.lock.release (hunk 3 above)
Comment (suyograo) [inline Gemfile.jruby-1.9.lock.release]: yep

File: Gemfile.jruby-1.9.lock.release (hunk 4 above)
Comment (suyograo) [inline Gemfile.jruby-1.9.lock.release]: yep

**Comment by suyograo** [comment]
All of these dependencies look ok to me.

**Comment by colinsurprenant** [comment]
ok, good, LGTM!

**Pull request merged**